packages/libs/error-stack/src/ext/stream.rs

```diff
@@ -0,0 +1,361 @@
+//! Implements support for `error-stack` functionality for streams. The main trait in this module
+//! is [`StreamReportExt`] (which is so named to avoid conflict with other stream-related types
+//! common to the Rust `async` ecosystem).
+use core::{
+    fmt::{Debug, Display},
+    task::Poll,
+};
+
+use futures_core::Stream;
+
+use crate::{Context, Report, ResultExt};
+
+// note: called this so as to not conflict with other `StreamExt`s
```

Comment:
Good call. However, we already have that in the crate docs so I think this line can be removed.

```suggestion
```